docs/tutorials/on-device-training.md

```diff
@@ -0,0 +1,1406 @@
+---
+title: On-Device Training
+nav_order: 11
+parent: Tutorials
+---
+
+# On-Device Training: Building an Android Application
+
```

Comment:
Add an Index to list all the sections to make it easy to navigate.

Reply:
This can be done automatically by the jekyll generator. Have a look at one of the other tutorials to see how this is done.